packages/nx-adsp/src/generators/angular-app/angular-app.ts

```diff
@@ -45,9 +45,11 @@ function normalizeOptions(
     nginxProxies
   };
 }
+
 function addFiles(host: Tree, options: NormalizedSchema) {
   const templateOptions = {
     ...options,
+    ...options.adsp,
```

Comment:
If `adsp` is a type I believe this will include the `adsp` details in both the root of the `templateOptions` as well as `templateOptions.adsp`. If `adsp` is just a string value then line 52 isn't needed

Reply:
This is the same as the react version of the app. When I didn't have this, it wasn't passing the interpolated options to the rest of the app
ie <%= appName %> wasn't being interpolated